build.gradle

```diff
@@ -40,25 +40,28 @@ apply plugin: 'com.palantir.external-publish'
 apply plugin: 'com.palantir.baseline'
 apply plugin: 'com.palantir.consistent-versions'
 apply plugin: 'com.palantir.git-version'
-apply plugin: 'com.palantir.java-format'
 apply plugin: 'com.palantir.revapi'
 
-group 'com.palantir.refreshable'
-version System.env.CIRCLE_TAG ?: gitVersion()
+allprojects {
```

Comment:
Would it make sense to bite the bullet and move the main impl to /refreshable, instead of /src? to align with the rest of the world

Reply:
Perhaps, it would conflict with other branches though, so I’d rather do that in isolation

Reply:
#140